File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Commons/EvaluateCollaborationModel.ts
import EvaluateCollaborationDocumentModel = require('App/Models/Commons/EvaluateCollaborationDocumentModel');

interface EvaluateCollaborationModel {
    DocumentModels: EvaluateCollaborationDocumentModel[];
    FromWorkflow: boolean;
}

export = EvaluateCollaborationModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Collaborations/EvaluateCollaborationModel.cs
using System.Collections.Generic;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Collaborations
{
    public class EvaluateCollaborationModel
    {
        public List<EvaluateCollaborationDocumentModel> DocumentModels { get; set; }
        public bool FromWorkflow { get; set; }
    }
}
